syntax/verilog_systemverilog.vim

```diff
@@ -187,8 +187,7 @@ for name in ['class', 'clocking', 'covergroup', 'function', 'interface',
     endif
 
     if verilog_systemverilog#VariableExists('verilog_quick_syntax')
-        execute 'syn match verilogStatement "'.s:region_start.'"'
-        execute 'syn match verilogStatement "'.s:region_end.'"'
+        execute 'syn keyword verilogStatement '.s:region_start.' '.s:region_end
```

Comment:
All tests are passing, but I will still take a look at some extra files just in case.
As far as I know, keywords conflict with match_start/end regular expressions, since you are working with that I assume that's where s**t hit the fan ;)